package.json

```diff
@@ -10,7 +10,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git@github.com:paypal/anemone-machina.git"
+    "url": "https://github.com/krakenjs/kraken-js.git"
```

Comment:
Sorry for the confusion. I just used the "kraken-js" repo's block as an example, but this would need to reflect repo URL of https://github.com/krakenjs/anemone-machina